### ISSUE TITLE
`4.0.2` contains no files

### ISSUE THREAD

**Issue by 10hendersonm** [danez/pirates#75]
The publish that just occurred is empty, causing errors when `require('pirates')`ing as the `main` `lib/index.js` cannot be found.

```bash
/node_modules/pirates $ ls -a
.  ..  LICENSE  README.md  index.d.ts  package.json
```

**Comment by 10hendersonm** [comment]
Eh, sorry, just saw the fix commits going on.

**Comment by danez** [comment]
Yeah i noticed too, is fixed now in 4.0.3. npm also didn't let me unpublish 10 seconds after release :/
